src/frontend/src/icp-eth/components/tokens/ManageTokens.svelte

```diff
@@ -99,11 +100,13 @@
 
 	let allTokensSorted: Token[] = [];
 	$: allTokensSorted = nonNullish(exchangesStaticData)
-		? sortTokens({
-				$tokens: allTokens,
-				$exchanges: exchangesStaticData,
-				$tokensToPin: $tokensToPin
-			})
+		? pinEnabledTokensAtTop(
```

Comment:
We are sorting the all list of displayed tokens, will ICP and Ethereum  always be listed first?

Reply:
yes, I maintained the idea that the ones that we decided to pin always (like ETH, ICP, BTC, etc) are always on top, even if disabled (that is not the case for now, since we force the user to have them enabled, without a choice to disable)

Reply:
You might want to ask Artem but, as a user, if the enabled token are pined at the top, I would except the same logic as currently but, with enabled first and disabled then - i.e. the list of enabled tokens following the sorting rules and then the disabled tokens sorted by the rules.

But again, if that's an interesting feedback, better ask Artem.

Reply:
I asked Artem and he says that for now that is ok. In the future, when and if it is decided that those tokens can be disabled, we can check which decision to take:

- Leave them above the others
- Mix them with the others

So, for now, I would say, this is the simplest solution to have the enabled ones above the disabled.

Reply:
Oki doki. Thanks for double checking.